lib/suse/connect/connection.rb

```diff
@@ -66,6 +66,9 @@ def json_request(method, path, params = {})
           body: body,
           success: response.is_a?(Net::HTTPSuccess)
         )
+
+      rescue Zlib::Error
+        raise SUSE::Connect::NetworkError, 'Network error. Try again and report a bug if it keeps happening.'
```

Comment:
`Check your network connection and try again. If it keeps failing, report a bug.` better?